stable/spinnaker/templates/configmap/service-settings.yaml

```diff
@@ -31,3 +31,9 @@ data:
       useExecHealthCheck: false
       serviceType: NodePort
     {{- end }}
+{{- if  and .Values.halyard.additionalServiceConfigMaps.create .Values.halyard.additionalServiceConfigMaps.data }}
```

Comment:
Sorry, but the variable name `additionalServiceConfigMaps` looks a bit misleading. Unlikely `additionalProfileConfigMaps`, this one doesn't create any new ConfigMap objects and just extends the existing one. 

Let's maybe use something like `additionalServiceSettings.extraData`, so it will be compatible with #13049 ?